src/components/Addbook.js

```diff
@@ -1,14 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/bookSlice';
 import './Addbook.css';
+import Books from './Books';
 
-const Addbook = () => (
-  <div className="new_book">
-    <form>
-      <input type="text" placeholder="title" />
-      <input type="text" placeholder="author" />
-      <button type="submit">Add Book</button>
-    </form>
-  </div>
-);
+const Addbook = () => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    dispatch(addBook({ title, author }));
+    setTitle('');
+    setAuthor('');
+  };
+
+  return (
+    <>
+      <Books />
+      <div className="new_book">
+        <form onSubmit={handleSubmit}>
+          <input
+            type="text"
+            placeholder="title"
+            value={title}
+            onChange={(e) => setTitle(e.target.value)}
+          />
+          <input
+            type="text"
+            placeholder="author"
+            value={author}
+            onChange={(e) => setAuthor(e.target.value)}
+          />
+          <button type="submit">Add Book</button>
+        </form>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job for adding all changes requested by the previous reviewer, however I think it would be better if you add a `required` attribute in all of your `input` to prevent a `user` a new book and for validation reasons.